common/app/conf/switches.scala

```diff
@@ -572,6 +572,24 @@ object Switches {
     exposeClientSide = true
   )
 
+  val TopAboveNavAdSlot728x90Switch = Switch(
+    "Commercial",
+    "top-above-nav-728-90",
+    "Expect a 728 x 90 ad in top-above-nav slot on UK network front.",
+    safeState = Off,
+    sellByDate = new LocalDate(2015, 7, 22),
+    exposeClientSide = true
```

Comment:
Hmm that does kinda defeat the point of this being here, though, which is to indicate to people looking at switches where they're used, and also to reduce the amount of stuff outputted into the page object.